src/NavBrand.js

```diff
@@ -0,0 +1,74 @@
+import React, { cloneElement } from 'react';
+import BootstrapMixin from './BootstrapMixin';
+import classNames from 'classnames';
+
+import createChainedFunction from './utils/createChainedFunction';
+
+const NavBrand = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    navbar: React.PropTypes.bool,
```

Comment:
Well, it's used for something like: https://github.com/react-bootstrap/react-bootstrap/blob/1173725f31d6c1aad34cffab74039fa790cc2042/src/Nav.js#L80. The prop doesn't do anything else otherwise.